workflow-examples/pom.xml.orig

```diff
@@ -0,0 +1,237 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Why do we need this file? I see some rebase conflict markings

Reply:
Ops! Thanks for catching this! I'll fix it